core/src/apps/webauthn/knownapps.py.mako

```diff
@@ -39,7 +43,8 @@ def by_rp_id_hash(rp_id_hash: bytes) -> FIDOApp | None:
         # ${type} key for ${app.name}
         return FIDOApp(
             label=${black_repr(label)},
-            icon=${black_repr(app.icon_res)},
+            icon=${black_repr(app.icon_res)},  # UI1, TO BE REMOVED
```

Comment:
It would be possible to do this trick for `UI1`, but I think it is not needed, as very soon we will get rid of `UI1` - probably this code will never even be released with `UI1`.

Reply:
OK, then we need to keep in mind not to merge this PR before the upcoming release is forked off.

Reply:
Done in [f44f512](https://github.com/trezor/trezor-firmware/pull/2556/commits/f44f5126ddf39a27ba035f698f2512abf0ea72c8) - it was just one formatted string in the end